### PR TITLE
Updated documents for 2.11.4

### PR DESCRIPTION
**Pull request by ktabata**

### CODE REVIEW

File: doc/reference.md [hunk 1]
```diff
@@ -694,10 +694,10 @@ Shows two levels of two options, four in total.
 ## @video
 
 This command plays a video file.
-At the moment, this functionality is enabled on Windows only.
+At the moment, this functionality is enabled on Windows and Mac only.
 
-Recommended video file format is `.WMV` file because of royalty fee.
-It is not recommended, but `.AVI` file which contains `H.264` + `AAC` is also supported on Windows 10/11.
+On Windows, please use `.wmv` video file format.
+On Mac, please use `.mp4` video file format.
```
Comment (KyouBrayden) [inline doc/reference.md:700]: Change to 'On Mac, please use the `.mp4` video file format.'

File: build/web-kit/readme-en.html [hunk 2]
```diff
@@ -101,6 +101,11 @@ <h2>Contributors</h2>
 
 <h2>Revisions</h2>
 <ul>
+  <li>Rev.48 23rd October 2022
+    <ul>
+      <li>Supported Suika 2.11.4</li>
```
[the remark this answers is not in the document]
Comment (ktabata) [reply build/web-kit/readme-en.html]: Thank you!

File: doc/reference.md [hunk 3]
```diff
@@ -694,10 +694,10 @@ Shows two levels of two options, four in total.
 ## @video
 
 This command plays a video file.
-At the moment, this functionality is enabled on Windows only.
+At the moment, this functionality is enabled on Windows and Mac only.
 
-Recommended video file format is `.WMV` file because of royalty fee.
-It is not recommended, but `.AVI` file which contains `H.264` + `AAC` is also supported on Windows 10/11.
+On Windows, please use `.wmv` video file format.
```
[the remark this answers is not in the document]
Comment (ktabata) [reply doc/reference.md]: Thank you!

File: doc/reference.md (hunk 1 above)
Comment (ktabata) [reply doc/reference.md]: Thank you!

**Review by KyouBrayden — APPROVED** [review]

**Pull request merged**